### PR TITLE
I2672 roles and responsibility

### LINKED ISSUE #2672: Update "Roles and Responsabilities" in the documentation
updating the "Roles and Responsibilities" before starting the new era of CEA.

### PR DESCRIPTION
**Pull request by shanshanhsieh**
This PR resolves #2672 
The Roles and Responsibilities are updated according to a meeting between me and @JIMENOFONSECA .
Meeting note: https://docs.google.com/document/d/1Dzmxej63BZz6Y3RQWHm4RUX73ja2R4b--Pp4ykOgAxM/edit

To test:
1. go to https://city-energy-analyst.readthedocs.io
2. switch branch to i2672 in the bottom left
3. review the page: Roles and Responsibilities 

Please leave comments below : )

### CODE REVIEW

File: docs/agile-with-CEA/activities.rst [hunk 1]
```diff
@@ -2,37 +2,37 @@ Activities
...
 
 Review event
 -------------
 
-Duration: 1 - 4 hours.
-Frequency: at the end of the sprint
-Scope: Present a realtime DEMO and receive feedback of the Product Sponsor.
+* Duration: 1 - 4 hours.
+* Frequency: at the end of the sprint
+* Scope: Present a realtime DEMO and receive feedback of the Product Sponsor.
```
Comment (JIMENOFONSECA) [inline docs/agile-with-CEA/activities.rst:28]: feedback of the product owner and customer team mainly. The product owner is supposed to be on top of 1. what is technically possible, what users need, what the priorities are, and what should be accepted and what not. The customer team can also give feedback if they are there.

File: docs/agile-with-CEA/Events.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+===================================
+
+Frequency
+---------
+This event should be scheduled **quarterly**.
+
+
+Attendees
+---------
+While this event is mainly an update to the product sponsors, it is also open to the entire CEA team.
```
Comment (JIMENOFONSECA) [inline docs/agile-with-CEA/Events.rst:11]: including the customer team (those individuals that are constantly using CEA)

File: docs/agile-with-CEA/activities.rst [hunk 3]
```diff
@@ -2,37 +2,37 @@ Activities
...
 Attendees: Scrum Master, Product Owner, Product Sponsor, and Development Team.
 
 Retrospective
 -------------
 
-Duration: 1 hour.
-Frequency: After every review event
-Scope: Discuss what was wrong and set next steps to follow.
+* Duration: 1 hour.
+* Frequency: After every review event
```
Comment (JIMENOFONSECA) [inline docs/agile-with-CEA/activities.rst:35]: Perhaps this is called by the PO or SM on a less frequent basis when the team needs to talk about how good they are working and practices. Time to blow off some steam too.

File: docs/agile-with-CEA/roles-and-responsibilities.rst [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+
+Scrum Master (SM)
+-----------------
+
+The Scrum Master is the servant leader and maintainer of agile practices.
+
+Responsibilities
+****************
+#. Guarantee the success of our processes by maintain all agile practices (including :doc:`activities`).
```
Comment (JIMENOFONSECA) [inline docs/agile-with-CEA/roles-and-responsibilities.rst:81]: let's make this explicit, the SM (@daren-thomas ) should coordinate the activities including sprint planning, development, review and retrospective. This is based on standard practice.

PO (@shanshanhsieh ) should coordinate the roadmap meeting and any other meeting with customers, e-mail channels, website etc. 

It is better that coordination activities are distributed in this way so neither of the SM or Po is overloaded in admin, yet they can take a leading role and commit to it.

File: docs/agile-with-CEA/activities.rst (hunk 3 above)
Comment (shanshanhsieh) [reply docs/agile-with-CEA/activities.rst:35]: i think it is still good to have a defined structure and frequency.
and i think there are always things to discuss in retropect!

File: docs/agile-with-CEA/activities.rst (hunk 1 above)
Comment (shanshanhsieh) [reply docs/agile-with-CEA/activities.rst:28]: i think we can replace this one with the SM & PO meeting that happens one day before the Planning event.
there, we discuss what goes into the product backlog, set priorities, and discuss the users need. 

and the original description is what we will do in the Roadmap Planning and Review Event.

**Comment by shanshanhsieh** [comment]
Hi @daren-thomas ,
I've resolved most of the comments from @JIMENOFONSECA.
You can still see Jimeno and my inputs to `docs/agile-with-CEA/activities.rst `, i will leave the final call to you.
Please go ahead and shape the agile practice for CEA : )

**Comment by daren-thomas** [comment]
@shanshanhsieh I have updated the activities page. Would you mind reviewing that for me please?

**Comment by shanshanhsieh** [comment]
hello @daren-thomas ,
the updates look good!
one thing that is not clear is the *Review Event* in Activities, are you referring to the same thing as the [Roadmap Planning and Review Event](https://city-energy-analyst.readthedocs.io/en/i2672-roles-and-responsibility/agile-with-CEA/Events.html)?

**Pull request merged**